### ISSUE TITLE
[NVDA] A11y_MicrosoftGuides_Outline_RearrangeSteps_Screenreader: NVDA doesn’t announce ‘Popup window’ when it opens.

### ISSUE THREAD

**Issue by ShwetaBaglane** [nvaccess/nvda#10055]
#### Logon Details as below:
Username: guidesselfhost@M365x699413.onmicrosoft.com
Instance: Test Test MRWNorthGuides Helf-host

### Steps to reproduce:
1. Open the Microsoft Dynamics 365 Guides (Preview) desktop application.
2. Sign-In into application using correct credentials.
3. Select the instance and click on Continue.
4. Navigate to any existing guides under the recent tab.
5. Navigate to any step within any task using tab key.
6. Start screen reader as NVDA.
7. Right Click on it (as currently not accessible by keyboard) to open the popup window.

### Actual behavior:
While NVDA is on and navigate to any step within any task. When pressed right click on it, popup window opens but screen reader doesn’t announce as ‘Popup window’ when it opened.

Same issue is also observed on "Add step" & "Task Options" scenario. When we right click on Add Step button or click on Task options, a pop up opens up which is shown in NVDA speech viewer but is not announce by NVDA. Screenshot attached for reference.

### Expected behavior:
While NVDA is on and navigate to any step within any task. When pressed right click on it, popup window opens and screen reader should announce as ‘Popup window’ when it is opened

### Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

### User Impact:
Screen reader only users will face difficulty as they will not get the information that popup window is opened.

### MAS Reference:
MAS1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

#### NVDA version:
NVDA version: 2018.3.2
#### Windows version:
Win10 Enterprise 1903
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by DrSooom** [comment]
@ShwetaBaglane: Same comment as in issue 10054:
> Please try to reproduce this with [NVDA 2019.2 RC2](https://ci.appveyor.com/api/buildjobs/qqofdhd6mlh8mq5u/artifacts/output/nvda_2019.2rc2.exe) (running via the Installer should be enough) and run NVDA before starting Microsoft Dynamics 365 Guides. Testing this with Narrator wouldn't be bad too. Thanks.

**Comment by Brian1Gaff** [comment]
I'd also suggest that nvda really should be installed since some very odd 
effects occur in Windows 10 on temp copies in my experience.
 Obviously for things to be recognised correctly nvda has to be running 
before anything is entered as this is how most people will use it, you can 
after all turn off the speech if you don't want it at that moment. Also of 
course screen shots are not terribly helpful as  many of the coders are 
blind. Debug logs might be more useful.
 Brian
original Message -----

**Comment by ShwetaBaglane** [comment]
Issue still repro on NVDA 2019.2 RC2 version. Here I want to add- Speech Viewer displays text as popup Window but the same is not announced by NVDA.
Narrator announces correctly as 'popup window' when it opens.

**Comment by brunasilvalopes** [comment]
This bug can also be seen on the Microsoft's publicly available app XAML Controls Gallery: https://www.microsoft.com/store/productId/9MSVH128X2ZT

**Steps to repro:**

1. Install and open XAML Controls App
2. On the home page, on the left menu, navigate to the option "Menus and Toolbars"
3. On the Menus and Toolbars Page, navigate to the Item MenuFlyout
4. On the MenuFlyout page, navigate to the first button called "Sort" 
5. Press Enter
6. Notice the bug: NVDA's Speech Viewer writes: 

- 7. "Popup window"
- 8. "By rating 1 of 3"

9. But NVDA only announces:

- 10. "By rating 1 of 3"

The UIA tree looks correct for the menu flyout:
**Screenshot:**
![image](https://user-images.githubusercontent.com/13021878/62907602-e2079500-bd28-11e9-8baf-aab6ed6c2208.png)

**Transcript of the screenshot:**
"XAML Controls Gallery" window
    "PopupHost" window​
        "Popup" window​
              "" menu​
                 "By rating" menu item​
                 "By match" menu item
                 "By distance" menu item

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
